src/gui_common/CustomWindow.cs

```diff
@@ -11,11 +12,38 @@
 /// </remarks>
 public class CustomWindow : Control
 {
+    [Export]
+    public Array<NodePath> WindowReorderingPaths = new();
```

Comment:
It looks like the reordering functionality is not implemented at this class level, so shouldn't all of these properties be put in `CustomDialog` instead as that's the level where these seem to be used?

Reply:
You are right. I wanted to have them here in case other children of this class needed them but I didn't think of that they would be doing nothing in this this class.

Reply:
I don't know why but after this change godot wants to save default values of those properties into scenes which means that changing the default values has no effect in the places where it has been saved already. Is it an issue that is just on my side or am I missing something?
![obrazek](https://user-images.githubusercontent.com/38793988/235952343-f2ec70e3-e3e9-4d17-ba6a-85c65cb3f5cb.png)

Reply:
Godot does sometimes nonsensical property saves. Usually helps to rebuild the code from Godot and then close and reopen the Godot editor after that it usually for me starts to correctly detect default values when saving scenes.

Reply:
I still can't get this to function properly no matter what I do. Even cloning Thrive again didn't fix it. I currently don't need to make any scene changes (I hope) so I will let it be for now.

Reply:
Once this is ready for playtesting, I'll investigate this myself.

Reply:
I think I've found the reason. The CustomDialog scene needed to have these default values put in it. Then they don't appear in every scene that derives from the dialogs.

I'm doing some fixing up on your PR and will commit and push my changes later today. If I end up being able to fix everything and playtest things I may end up merging this PR as well today.